app/models/ok2_explore/death.rb

```diff
@@ -1,5 +1,5 @@
 class Ok2Explore::Death < ApplicationRecord
-  belongs_to :county
+  belongs_to :county, optional: true
```

Comment:
When testing I made it about 1/5 of the way through the year and noticed a handful of records with no county. Went ahead and made nullable but that's in its own commit if we want to revert.